ci/jenkins/test-vmc.sh

```diff
@@ -244,6 +244,20 @@ function setup_cluster() {
     fi
 }
 
+function copy_image {
```

Comment:
Hello, it is good to have a `copy_image` function to make `deliver_antrea` clean. However, it confuses me that this function doesn't include judgement for "Coverage" while "centos-7" is included. I think these 2 parameters should be treated equally. So one parameter `IP` is enough for this function.
Could you please explain why you design like this? I just woke up and maybe my mind is not clear enough...

Reply:
@lzhecheng you make a good point. If the tests don't pass and we need to amend this PR, we will change this function as well as per your suggestion. If the tests do pass however, I suggest we merge this in order to make the release, and @srikartati can take as an action item to improve the function implementation. Does that work for you?

Reply:
"coverage" is used in deliver_antrea (L328-L333). @antoninbas identified a bug with image name when coverage is not true; it should be projects.registry.vmware.com and not docker.io. Therefore we considered 4 args.

Reply:
@lzhecheng is wondering if we can reduce the number of parameters by accessing `$COVERAGE` directly in the function implementation. However, thinking more about this, I think it would make the function implementation dependent on which image we are pushing (antrea or flow-aggregator), which is probably not great.

Reply:
Sure, we can improve CI after this PR is merged. I fully understand the pain to re-run all tests...

Reply:
@antoninbas sorry I didn't get your point. I think all logic of the loop in `deliver_antrea` will be in `copy_image`. And flow-aggregator image always gets delivered, right? "centos-7" and "coverage" take care of the original logic.

Reply:
I think the idea was to have a function that support uploading any image when invoked, rather than a function that takes care of uploading all images at once in a single invocation. With your proposal, won't we need to duplicate logic inside of the function for the antrea image and for the flow-aggregator image?

Reply:
Oh, now I get your point. Yes, you are right. Such function is the one needed and I just try to move as much as logic to other functions to make `deliver_antrea` clean.
Ideally, we can put a `copy_some_images` in `deliver_antrea` and `copy_image` in `copy_some_images` but it is unnecessary in our case.